src/DurableTask.Netherite.AzureFunctions/NetheriteProviderFactory.cs

```diff
@@ -46,18 +47,39 @@ public class NetheriteProviderFactory : IDurabilityProviderFactory
         public const string ProviderName = "Netherite";
         public string Name => ProviderName;
 
-        // Called by the Azure Functions runtime dependency injection infrastructure
+        /// <summary>
+        /// This constructor should not be called directly. It is here only to avoid breaking changes.
+        /// </summary>
+        [Obsolete]
```

Comment:
Obsolete'ing APIs has some differing viewpoints. .NETs policy is as follows:

> An API that shipped in a long-term support (LTS) release must be obsoleted in the subsequent LTS release before it can be removed. In rare cases, exceptions are made to obsolete an API before the subsequent LTS release based on business needs. All obsoletions are documented and communicated to customers.

It takes _two_ LTS releases to remove a public API! First one marks it obsolete, second removes. This is because _technically_ adding `[Obsolete]` can be viewed as a compilation breaking change if the project has `WarnAsError=true`.

Buuuut, I think in this case you are fine. This is just more of a fun FYI to express how "_fun_" maintaining public APIs is 😆